src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+                                    </MenuInitiaLetterCircle>
+                                    <MenuName>{UserName}</MenuName>
+                                    <NavLink to="login" onClick={`${toggle}${logOut}`}>
+                                        <ExitIconMobile />
+                                    </NavLink>
+                                </MenuHeaderBlock>
+                            </MobileHamburgerLoginBox>
+                            <MobileHamburgerCrossBox onClick={toggle}>
+                                <MobileHamburgerCross>
+                                    <MobileHamburgerCross1 />
+                                </MobileHamburgerCross>
+                            </MobileHamburgerCrossBox>
+
+                            <NavLink to="home" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenuUp>Главная</MobileHamburgerSectionMenuUp>
+                            </NavLink>
+                            <NavLink to="planning" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Планирование</MobileHamburgerSectionMenu>
+                            </NavLink>
+                            <NavLink to="awards" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Награды</MobileHamburgerSectionMenu>
+                            </NavLink>
+                            <NavLink to="contacts" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Контакты</MobileHamburgerSectionMenu>
+                            </NavLink>
+                        </MobileHamburgerbackground>
+                    </>
+                    }
+                </>
+            }
+        </>
+    )
+}
+export default Header
+
+const LogoHead = () => (
+    <NavLink to="home">
+        <LogoHeader>
+            KidsLike
+            <Logo />
+        </LogoHeader>
+    </NavLink>
+)
+const MenuAuthDeskTab = () => (
+    <>
+        <NavLink to="login"
+            style={({ isActive }) => { return { color: isActive ? 'black' : 'rgb(133, 133, 152)' } }}>
+            <MenuSection>Авторизация</MenuSection>
+        </NavLink>
+        <NavLink to="contacts"
+            style={({ isActive }) => { return { color: isActive ? 'black' : 'rgb(133, 133, 152)' } }}>
+            <MenuContact style={{ padding: '0px 0px 0px 16px' }} >Контакты</MenuContact>
+        </NavLink>
+    </>
+)
+const PointBalUser = () => (
+    <PointsBalance>
+        Баланс баллов:
+        <NumberBalance>{UserBalance}</NumberBalance>
```

Comment:
We have agreed to get rid of russian language from our project :)
Could you please translate all the titles to English?

Reply:
Thank you) I'm starting to fix